cmd/kev/cmd/render.go

```diff
@@ -91,18 +95,22 @@ func runRenderCmd(cmd *cobra.Command, _ []string) error {
 	singleFile, err := cmd.Flags().GetBool("single")
 	dir, err := cmd.Flags().GetString("dir")
 	envs, err := cmd.Flags().GetStringSlice("environment")
+	verbose, _ := cmd.Root().Flags().GetBool("verbose")
 
 	if err != nil {
 		return err
 	}
 
-	log.Infof("⚙️  Output format: %s", format)
+	reporter := getReporter(true)
+	if !verbose {
+		log.SetOutput(ioutil.Discard)
+	}
 
+	_, _ = reporter.Write([]byte("✓ Render\n"))
 	if err := kev.Render(format, singleFile, dir, envs); err != nil {
 		return err
 	}
-
-	log.Info("🧰 App render complete!")
+	_, _ = reporter.Write([]byte(fmt.Sprintf(" → Rendering manifests in %s output format ... Done\n", strings.Title(format))))
```

Comment:
When in "regular" mode this obviously missing some context e.g. which environment was rendered.

Reply:
Good point - need re-think this.